gdnative-sys/build.rs

```diff
@@ -179,11 +176,12 @@ mod header_binding {
 
         let target_os = std::env::var("CARGO_CFG_TARGET_OS").unwrap();
         let target_arch = std::env::var("CARGO_CFG_TARGET_ARCH").unwrap();
+        let target_vendor = std::env::var("CARGO_CFG_TARGET_VENDOR").unwrap();
 
         // Workaround: x86_64 architecture is unsupported by the iPhone SDK, but cargo-lipo will
         // try to build it anyway. This leads to a clang error, so we'll skip the SDK.
```

Comment:
Looks like it's working now. Great! Now we can remove this `Workaround` comment since it no longer applies. After that it should be good to merge!

Thanks a lot!